test/libethereum/TransactionTests.cpp

```diff
@@ -63,7 +63,7 @@ void doTransactionTests(json_spirit::mValue& _v, bool _fillin)
 			try
 			{
 				Transaction txFromFields(rlpStream.out(), CheckTransaction::Everything);
-				if (!txFromFields.signature().isValid())
+				if (!txFromFields.signature().isValid() && !txFromFields.signature().isZeroSignature())
```

Comment:
I think we can put `isZeroSignature()` check inside `isValid()`, then we won't have to duplicate it three times (here and in `TransactionBase` constructor)